sql/patchdemo.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE IF NOT EXISTS patchdemo COLLATE "utf8_general_ci";
```

Comment:
Why `COLLATE "utf8_general_ci"`? I would just remove this part and use the default.

I might be out of date on this, but IIRC in MySQL, `utf8mb4` is the real Unicode, while `utf8` is BMP-only Unicode and it will explode in your face when you try to use non-BMP characters like emojis. Not sure how exactly a collation might explode, but still.

It seems that the default is `utf8mb4_0900_ai_ci` (https://dev.mysql.com/doc/refman/8.0/en/charset-applications.html), no idea what exactly it does, but it sounds reassuring.

Reply:
Sure - I'm just copying from old stuff, but if the defaults are sensible that's fine.

Reply:
Fixed